guides/Web_Frameworks/CakePHP.md

```diff
@@ -0,0 +1,47 @@
+# CakePHP
+
+CakePHP is a MVC Framework written in PHP. The major difference between 3.0 and 2.0 is 3.0 is object oriented (uses class and objects).
+It has lots of features which can be used in fast development with less code and build complex applications faster than plain PHP.
```

Comment:
1) After, 2.0 please write - version of CakePHP.
2) Change "build" to "builds".